src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/VideoLinkMigrationController.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.whereabouts.controllers
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.whereabouts.services.court.VideoLinkMigrationService
+
+@Tag(name = "court")
```

Comment:
`court` or `migrate`?

Just wondering if perhaps `/migrate/video-link-bookings/{videoBookingId}`

Reply:
Yes - it would take it out of the group daily-use endpoints in /court too. I'll do it.

Reply:
Now changed to a 'Migration' controller, with endpoint `/migrate/video-link-booking/{videoBookingId}` - please note the singular video-link-booking.

Reply:
I'll expand this controller with the triggering endpoint next.

Reply:
ta